downloads/index.html

```diff
@@ -1,166 +1,109 @@
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Open+Sans:wght@800&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:wght@300"
-      rel="stylesheet"
-    />
 
-    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
-    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
-    <link rel="manifest" href="/site.webmanifest" />
-    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
-    <meta name="msapplication-TileColor" content="#15c666" />
-    <meta name="theme-color" content="#02d663" />
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <meta http-equiv="cache-control" content="no-cache" />
+  <meta http-equiv="expires" content="Mon, 22 Jul 2002 11:12:01 GMT" />
+  <title>Lawnchair Downloads</title>
+  <meta name="description" content="Download the latest releases of Lawnchair, alongside it's plugins" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+    integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Open+Sans:wght@800&display=swap"
+    rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:wght@300" rel="stylesheet" />
 
-    <link href="/assets/common.css?qwertyuiop" rel="stylesheet" />
-    <link href="/assets/downloads.css?qwertyuiop" rel="stylesheet" />
-  </head>
-  <body>
-    <header>
-      <a href="/" class="lawnchair__logo">
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+  <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+  <link rel="manifest" href="/site.webmanifest" />
+  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
+  <meta name="msapplication-TileColor" content="#15c666" />
+  <meta name="theme-color" content="#02d663" />
+
+  <link href="/assets/common.css?qwertyuiop" rel="stylesheet" />
+  <link href="/assets/downloads.css?qwertyuiop" rel="stylesheet" />
```

Comment:
Could you change the `?qwertyuiop` text to something else? The browser caches the CSS, causing some styling problems, so changing this forces the browser to redownload the CSS.

Something similar to:
```suggestion
  <link href="/assets/common.css?asdfghjkl" rel="stylesheet" />
  <link href="/assets/downloads.css?asdfghjkl" rel="stylesheet" />
```

This applies to the other html pages.